GPU/Common/CMakeLists.txt

```diff
@@ -0,0 +1,52 @@
+cmake_minimum_required(VERSION 2.8.12 FATAL_ERROR)
+
+if(ALIGPU_BUILD_TYPE STREQUAL "O2")
+    set(MODULE O2AliGPUCommon)
+elseif(ALIGPU_BUILD_TYPE STREQUAL "ALIROOT")
+    set(MODULE AliGPUCommon)
+endif()
+
+set(SRCS
+    AliGPUCommon.cxx
+)
+
+set (HDRS
+	AliGPUCommonDef.h
+	AliGPUCommonDefGPU.h
+    AliGPUCommonMath.h
+    AliGPUCommonFairLogger.h
+    AliGPUCommonRtypes.h
+    AliGPUCommonTransform3D.h
+	FlatObject.h
+)
+
```

Comment:
@davidrohr could you add a LinkDef file with at least 
``#pragma link C++ class ali_tpc_common::Base::FlatObject+;``, to allow root persistency for derived objects.
BTW, should not the namespace become gpu_common?